fossology/fossology
# feat(licenseref): convert licenseref from sql to json format

## Description

Change licenseref.sql to licenseref.json.

### Changes

1. parse JSON file and insert into database

## How to test

1. For a existing fossology instance pull `contrib/featLicenseRefSQLtoJSON` and run the `postinstall` scripts.
2. Check database for all the entries.
3. Upload a file and run monk for sample data.
4. Delete database and run a fresh install.
5. Check database for all the entries.

Closes #1286
